src/files/BrsFile.ts

```diff
@@ -77,6 +77,27 @@ export class BrsFile {
         this.srcPath = value;
     }
 
+    /**
+     * Should this file be written to disk
+     */
+    public get isPublishable() {
+        let isPublishable = false;
+        this.ast.walk(createVisitor({
+            FunctionStatement: () => {
+                isPublishable = true;
+            },
+            MethodStatement: () => {
```

Comment:
Methods can't exist on their own, so we can probably remove this one.

Reply:
removed 👍